<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(38, 38, 38);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>871</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">display: flex;
align-items: center;
justify-content: center;
color: rgb(0, 255, 0);
font: 50px &quot;Lucida Console&quot;;</string>
   </property>
   <property name="text">
    <string>Grassfire Algorithm Simulator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">display: flex;
align-items: center;
justify-content: center;
color: rgb(0, 255, 0);
font: 20px &quot;Lucida Console&quot;;</string>
   </property>
   <property name="text">
    <string>By Callum Brezden</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">display: flex;
align-items: center;
justify-content: center;
color: rgb(0, 255, 0);
font: 20px &quot;Lucida Console&quot;;</string>
   </property>
   <property name="text">
    <string>Grid Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">display: flex;
align-items: center;
justify-content: center;
color: rgb(0, 255, 0);
font: 20px &quot;Lucida Console&quot;;</string>
   </property>
   <property name="text">
    <string>Grid Height</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gridWidthText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gridHeightText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="computeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Lucida Console&quot;;</string>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
